platform/hackerrank/src/main/java/com/learning/JavaList.java

```diff
@@ -0,0 +1,56 @@
+package com.learning;
+
+import static java.util.Arrays.*;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+class JavaList {
+
+  private static String command;
+  private static String indexValueLine;
+  private static int index;
+  private static int value;
+
+  private static void scanInput(Scanner scanner) {
+    command = scanner.nextLine();
+    indexValueLine = scanner.nextLine();
+
+    index = Integer.parseInt(indexValueLine.split(" ")[0]);
+    value = Integer.parseInt(indexValueLine.split(" ")[1]);
+  }
+
+  private static void executeCommands(List<Integer> list) {
+    switch (command) {
+      case "INSERT":
+        if (index >= list.size()) {
+          list.add(index, value);
+        } else {
+          list.set(index, value);
+        }
+        break;
+      case "DELETE":
+        list.remove(index);
```

Comment:
Would be nice to have a `default` case as well:
```suggestion
      case "DELETE":
        list.remove(index);
      default:
            System.out.println("Unknown command: " + command);  
```